tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -275,3 +279,66 @@ TEST(RDFHelpers, SaveGraphToFile)
 
    gSystem->Unlink(outFileName);
 }
+
+TEST(RunGraphs, RunGraphs)
+{
+#ifdef R__USE_IMT
+   ROOT::EnableImplicitMT();
+#endif // R__USE_IMT
+
+   ROOT::RDataFrame df1(3);
+   auto df1a = df1.Define("x", "float(rdfentry_)");
+   auto r1 = df1a.Sum("x");
+   auto r2 = df1a.Count();
+
+   ROOT::RDataFrame df2(3);
+   auto df2a = df2.Define("x", "2.f * float(rdfentry_)");
+   auto r3 = df2a.Sum("x");
+   auto r4 = df2a.Count();
+
+   std::vector<RResultHandle> v = {r1, r2, r3, r4};
+   ROOT::RDF::RunGraphs(v);
+
+   EXPECT_EQ(df1.GetNRuns(), 1u);
+   EXPECT_EQ(df2.GetNRuns(), 1u);
+
+   for (auto& h : v) EXPECT_TRUE(h.IsReady());
+   EXPECT_EQ(r1.GetValue(), 3.f);
+   EXPECT_EQ(r2.GetValue(), 3u);
+   EXPECT_EQ(r3.GetValue(), 6.f);
+   EXPECT_EQ(r4.GetValue(), 3u);
+}
+
+TEST(RunGraphs, EmptyListOfHandles)
+{
+#ifdef R__USE_IMT
+   ROOT::EnableImplicitMT();
+#endif // R__USE_IMT
+
+   testing::internal::CaptureStderr();
+   ROOT::RDF::RunGraphs({});
+   const std::string output = testing::internal::GetCapturedStderr();
+   const std::string expected = "Warning in <RunGraphs>: Got an empty list of handles\n";
+   EXPECT_STREQ(output.c_str(), expected.c_str());
```

Comment:
here and below, we have `ROOT_EXPECT_WARNING` and friends for this now

Reply:
Done